mtp_cashbook/templates/disbursements/base.html

```diff
@@ -13,7 +13,7 @@
         {% include 'proposition-tab.html' with view_name='disbursements:pending_list' subview_names='disbursements:pending_list disbursements:pending_detail disbursements:confirmed disbursements:update_sending_method disbursements:update_prisoner disbursements:update_amount disbursements:update_recipient_contact disbursements:update_recipient_bank_account' link_text=_('Confirm payments') suffix=confirm_tab_suffix %}
         {% include 'proposition-tab.html' with view_name='disbursements:search' subview_names='disbursements:search' link_text=_('Payments made') %}
         {% include 'proposition-tab.html' with view_name='disbursements:paper-forms' subview_names='disbursements:paper-forms' link_text=_('Paper forms') %}
-        {% include 'proposition-tab.html' with view_name='disbursements:submit_ticket' subview_names='disbursements:submit_ticket disbursements:feedback_success disbursements:process-overview' link_text=_('Get help') %}
+        {% include 'proposition-tab.html' with view_name='disbursements:process-overview' subview_names='disbursements:process-overview disbursements:feedback_success disbursements:process-overview' link_text=_('Get help') %}
```

Comment:
have a look at the change you've made in this line… what's wrong with it?

Reply:
Inside the subview_name I also need to change disbursements:process-overview to  disbursements:submit_ticket. Or maybe I didn't need to change anything in the subview_name. Just the order perhaps?